src/main/java/org/parabot/environment/api/utils/PaintUtil.java

```diff
@@ -0,0 +1,51 @@
+package org.parabot.environment.api.utils;
+
+import java.text.DecimalFormat;
+
+/**
+ * Functions frequently used for displaying data on paints
+ *
+ * @author AlexanderBielen
+ */
+public class PaintUtil {
+
+    /**
+     * Calculates how much given variable has gained per hour
+     *
+     * @param currentAmount total gained amount
+     * @param start         time from which to start counting in milliseconds
+     * @return              rate per hour
+     */
+    public static long calculatePerHour(int currentAmount, long start) {
+        return calculatePerHour(currentAmount, 0, start);
+    }
+
+    /**
+     * Calculates how much given variable has gained per hour, with variable start amount
+     *
+     * @param currentAmount total gained amount
+     * @param startAmount   start amount
+     * @param start         time from which to start counting in milliseconds
+     * @return              rate per hour
+     */
+    public static long calculatePerHour(int currentAmount, int startAmount, long start) {
+        return (int)(((double)(currentAmount - startAmount) * 3600000D) / (double)(System.currentTimeMillis() - start));
+    }
+
+    /**
+     * Formats a time difference from given timestamp till present into readable hh:mm:ss format
+     *
+     * @param start start time in milliseconds
+     * @return      readable timestamp
+     */
+    public static String formatRunTime(long start) {
```

Comment:
Might wanna check out the Timer class for this, as it somewhat has the same code. Maybe merge code to avoid duplicated code.

Reply:
That's stupid I completely missed the Timer class..
I'll see what I can merge!